source/adios2/engine/ssc/SscWriter.cpp

```diff
@@ -120,6 +138,233 @@ void SscWriter::SyncMpiPattern()
         std::cout << "SscWriter::SyncMpiPattern, World Rank " << m_WorldRank
                   << ", Writer Rank " << m_WriterRank << std::endl;
     }
+
+
+    std::vector<int> lrbuf;
+    std::vector<int> grbuf;
+    int m_AppID = 0;
```

Comment:
Same as above, these should be in the header file

Reply:
> Same as above, these should be in the header file

All the member variables are moved to the header file now.